crates/egui_demo_lib/src/demo/window_options.rs

```diff
@@ -6,7 +6,6 @@ pub struct WindowOptions {
     closable: bool,
     collapsible: bool,
     resizable: bool,
-    constrain: bool,
```

Comment:
why was the constrain demo removed?

Reply:
I probably made a mistake using git, it is unintentional, I restored it.